src/Tempest/Http/src/Routing/Construction/RouteConfigConstructor.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Http\Routing\Construction;
+
+use Tempest\Container\Singleton;
+use Tempest\Http\Route;
+use Tempest\Http\RouteConfig;
+
+/**
+ * @internal
+ */
+#[Singleton]
+final class RouteConfigConstructor
```

Comment:
This is great, I'm only a little in doubt about the name. Not a dealbreaker btw, but maybe something like `RouteConfigurator` would fit?

Reply:
Fixed, great naming suggestion